development/tools/cmd/diskscollector/README.md

```diff
@@ -0,0 +1,51 @@
+# Disks Garbage Collector
+
+## Overview
+
+This command finds and removes orphaned disks created in a Google Cloud Platform (GCP) project by kyma-gke-integration jobs.
+When kyma-gke-integration job installs Kyma in the GKE cluster, there are some disks created automatically by GCP.
+Usually all such disks are cleaned up by the job that provisioned the cluster.
+Sometimes though, the job is terminated before it's clean-up is finished.
+This causes a resource leak that is generating unwanted costs.
+Garbage collector finds and removes such disks.
+There are three conditions used to find "garbage" disks:
+- Disk name pattern (specific for the kyma-gke-integration job)
+- Disk users count (zero means the disk is unused)
+- Disk CreationTimestamp value (used to find disks existing for at least some configured number of hours)
+
+Disks that satisfy the above conditions are subject to removal.
```

Comment:
```suggestion
Disks that meet these conditions are subject to removal.
```

Reply:
We avoid words like "above" or "below", and as for "meet" instead of "satisfy" - it is just more common.